squarelet/users/fe_api/viewsets.py

```diff
@@ -0,0 +1,36 @@
+# Django
+from django.db.models import Prefetch
+
+# Third Party
+from allauth.account.models import EmailAddress
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+# Squarelet
+from squarelet.organizations.models import Membership
+from squarelet.users.fe_api.serializers import UserSerializer
+from squarelet.users.models import User
+
+
+class UserViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = User.objects.prefetch_related(
+        Prefetch(
+            "memberships", queryset=Membership.objects.select_related("organization")
+        ),
+        Prefetch(
+            "emailaddress_set",
+            queryset=EmailAddress.objects.filter(primary=True),
+            to_attr="primary_emails",
+        ),
+    ).order_by("created_at")
+    serializer_class = UserSerializer
+    permission_classes = (IsAuthenticated,)
+    lookup_field = "individual_organization_id"
+    swagger_schema = None
+
+    def get_queryset(self):
+        # Use the get_viewable() method from MembershipQuerySet
```

Comment:
I am not sure why you are using the `Membership` `get_viewable` here.  I believe we want to show all users to everyone?  If not, we should make a `User` `get_viewable` method with the proper logic.